pylint/test/extensions/test_check_return_docs.py

```diff
@@ -0,0 +1,234 @@
+"""Unit tests for the return documentation checking in the
+`DocstringChecker` in :mod:`pylint.extensions.check_docs`
+"""
+from __future__ import division, print_function, absolute_import
+
+import unittest
+
+import astroid
+from astroid import test_utils
+from pylint.testutils import CheckerTestCase, Message, set_config
+
+from pylint.extensions.check_docs import DocstringChecker
+
+
+class DocstringCheckerReturnTest(CheckerTestCase):
+    """Tests for pylint_plugin.RaiseDocChecker"""
+    CHECKER_CLASS = DocstringChecker
+
+    def test_ignores_no_docstring(self):
+        return_node = test_utils.extract_node('''
+        def my_func(self):
+            return False #@
+        ''')
+        with self.assertNoMessages():
+            self.checker.visit_return(return_node)
+
+    @set_config(accept_no_return_doc=False)
```

Comment:
`test_ignores_no_docstring` (above) does this already. Or would you like me to add a test with it set explicitly?